docs/container-model.c4

```diff
@@ -92,6 +92,40 @@ workspace {
 			timecardService -> singlePageApplication "return success or failure"
```

Comment:
Is there a way to review c4 files without trying to read the markup?

Reply:
Copy and paste C4 file content into https://structurizr.com/dsl